infra/chromadb_compaction/lambdas/stream_processor.py

```
@@ -559,31 +608,23 @@ def _parse_entity(
             available_fields=list(image.keys()) if image else "None",
+            complete_item_fields=(
+                list(complete_item.keys())
+                if "complete_item" in locals()
+                else "Not created"
+            ),
             pk=pk,
             sk=sk,
+            raw_complete_item=(
+                complete_item if "complete_item" in locals() else "Not created"
+            ),
         )
```

Ruff (0.13.1)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)